config/client.go

```diff
@@ -55,7 +55,7 @@ func GenerateClientConfig(c *model.Config, telemetryID string, license *model.Li
 	// This setting is only temporary, so keep using the old setting name for the mobile and web apps
 	props["ExperimentalEnablePostMetadata"] = "true"
 
-	props["EnableAppBar"] = strconv.FormatBool(*c.ExperimentalSettings.EnableAppBar)
+	props["DisableAppBar"] = strconv.FormatBool(*c.ExperimentalSettings.DisableAppBar)
```

Comment:
I don't think it matters for this because mobile probably doesn't use this at all, but in the past, we've left the old client config variable name just for compatibility reasons. That lets old clients keep working with the new setting
```go
	props["EnableAppBar"] = !strconv.FormatBool(*c.ExperimentalSettings.DisableAppBar)
	props["DisableAppBar"] = strconv.FormatBool(*c.ExperimentalSettings.DisableAppBar)
```